### PR TITLE
Updated hub page and doc.

### PR DESCRIPTION
**Pull request by jm-trd-ms**
Updated hub page and doc.  

https://docs.microsoft.com/microsoft-edge/index

### CODE REVIEW

File: microsoft-edge/toc.yml [hunk 1]
```diff
@@ -364,23 +362,23 @@
     - name: Extensions (Chromium)
       href: extensions-chromium/index.md
       items: 
-        - name: Getting Started
+        - name: Getting started
           items: 
             - name: Extension architecture
               href: extensions-chromium/getting-started/index.md
-            - name: Create an extension tutorial Part 1
+            - name: Create an extension tutorial part 1
```
Comment (Reezaali) [inline microsoft-edge/toc.yml:369]:
```suggestion
            - name: Create an extension tutorial (part 1)
```

File: microsoft-edge/toc.yml [hunk 2]
```diff
@@ -364,23 +362,23 @@
...
               href: extensions-chromium/getting-started/part1-simple-extension.md
-            - name: Create an extension tutorial Part 2
+            - name: Create an extension tutorial part 2
```
Comment (Reezaali) [inline microsoft-edge/toc.yml:371]:
```suggestion
            - name: Create an extension tutorial (part 2)
```

File: microsoft-edge/toc.yml [hunk 3]
```diff
@@ -364,23 +362,23 @@
...
               href: extensions-chromium/getting-started/part2-content-scripts.md
             - name: Sideload an extension
               href: extensions-chromium/getting-started/extension-sideloading.md
         - name: Developer Guide
           items: 
-            - name: Port Chromium Extension to Microsoft Edge
+            - name: Port Chromium extension to Microsoft Edge
```
Comment (Reezaali) [inline microsoft-edge/toc.yml:377]:
```suggestion
            - name: Port Chromium extensions to Microsoft Edge
```

File: microsoft-edge/toc.yml [hunk 4]
```diff
@@ -390,25 +388,25 @@
...
-            - name: Update Extension Listing
+            - name: Update extension listing
               href: extensions-chromium/publish/update.md
             - name: Submission states
               href: extensions-chromium/publish/submission-states.md
-            - name: Manage Account Settings
+            - name: Manage account settings
               href: extensions-chromium/publish/manage-settings.md
-        - name: Microsoft Edge Addons Policies
+        - name: Microsoft Edge addons policies
```
Comment (Reezaali) [inline microsoft-edge/toc.yml:397]:
```suggestion
        - name: Microsoft Edge Add-ons policies
```

File: microsoft-edge/toc.yml [hunk 5]
```diff
@@ -308,13 +306,13 @@
           href: devtools-guide/memory.md
         - name: Storage
           href: devtools-guide/storage.md
-        - name: Service Workers
+        - name: Service workers
           href: devtools-guide/service-workers.md
         - name: Emulation
           href: devtools-guide/emulation.md
-    - name: DevTools Protocol (Chromium)
+    - name: DevTools protocol (Chromium)
```
Comment (Reezaali) [inline microsoft-edge/toc.yml:313]:
```suggestion
    - name: DevTools Protocol (Chromium)
```

File: microsoft-edge/toc.yml [hunk 6]
```diff
@@ -308,13 +306,13 @@
...
       href: devtools-protocol-chromium.md
-    - name: DevTools Protocol (EdgeHTML)
+    - name: DevTools protocol (EdgeHTML)
```
Comment (Reezaali) [inline microsoft-edge/toc.yml:315]:
```suggestion
    - name: DevTools Protocol (EdgeHTML)
```

File: microsoft-edge/toc.yml [hunk 7]
```diff
@@ -390,25 +388,25 @@
...
+        - name: Microsoft Edge addons policies
           items: 
-            - name: Content Security Policy
+            - name: Content security policy
               href: extensions-chromium/store-policies/csp.md
-            - name: Edge Developer ADA Addendum
+            - name: Microsoft Edge developer ADA addendum
               href: extensions-chromium/store-policies/ada-addendum.md
-            - name: Microsoft Edge Addons Catalog Developer Policies
+            - name: Microsoft Edge addons catalog developer policies
```
Comment (Reezaali) [inline microsoft-edge/toc.yml:403]:
```suggestion
            - name: Microsoft Edge Add-ons catalog developer policies
```

File: microsoft-edge/toc.yml [hunk 8]
```diff
@@ -457,7 +455,7 @@
                   href: extensions/guides/packaging/localizing-extension-packages.md
                 - name: Using ManifoldJS to package extensions
                   href: extensions/guides/packaging/using-ManifoldJS-to-package-extensions.md
-                - name: Running the Windows App Certification Kit
+                - name: Running the Windows app certification kit
```
Comment (Reezaali) [inline microsoft-edge/toc.yml:458]:
```suggestion
                - name: Running the Windows App Certification Kit
```

File: microsoft-edge/toc.yml [hunk 9]
```diff
@@ -469,13 +467,13 @@
           href: extensions/troubleshooting.md
     - name: Hosting
       href: hosting/toc.md
-    - name: Origin Trials
+    - name: Origin trials
```
Comment (Reezaali) [inline microsoft-edge/toc.yml:470]:
```suggestion
    - name: Origin Trials
```

File: microsoft-edge/toc.yml [hunk 10]
```diff
@@ -469,13 +467,13 @@
...
       href: origin-trials/index.md
     - name: Performance analysis
       href: performance-analysis.md
-    - name: Privacy Whitepaper
+    - name: Privacy whitepaper
       href: privacy-whitepaper.md
-    - name: Progressive Web Apps (Chromium)
+    - name: Progressive web apps (Chromium)
```
Comment (Reezaali) [inline microsoft-edge/toc.yml:476]:
```suggestion
    - name: Progressive Web Apps (Chromium)
```

File: microsoft-edge/toc.yml [hunk 11]
```diff
@@ -484,9 +482,9 @@
           href: progressive-web-apps-chromium/offline.md
         - name: Service Workers
           href: progressive-web-apps-chromium/serviceworker.md
-        - name: Web App Manifests
+        - name: Web app manifests
           href: progressive-web-apps-chromium/webappmanifests.md
-    - name: Progressive Web Apps (EdgeHTML)
+    - name: Progressive web apps (EdgeHTML)
```
Comment (Reezaali) [inline microsoft-edge/toc.yml:487]:
```suggestion
    - name: Progressive Web Apps (EdgeHTML)
```

**Comment by jm-trd-ms** [comment]
#sign-off offline and merged.

**Pull request merged**